keras_nlp/samplers/beam_sampler_test.py

```diff
@@ -67,6 +68,23 @@ def test_stateful_call(self):
         )
         self.assertEqual(self.join_as_string(output), ["sequentially"])
 
+    def test_return_all_beams(self):
+        state_chars = list("sequentially")
+        state = tf.constant([[self.char_lookup[c] for c in state_chars]])
+        prompt = tf.fill((self.batch_size, self.length), self.char_lookup["z"])
+        output = self.sampler_all_beams(
```

Comment:
Let's use explicit names: `sorted_prompts` and `sorted_log_probs` here so that it's more clear to readers what we are testing.